Makefile

```
@@ -1,100 +1,146 @@
+# Setup
+# ------------------------------------------------------------------- #
 setup: ## Set up complete development environment
 	@echo "Setting up AstraDesk development environment..."
-	@./scripts/setup-dev-environment.sh
+	@if command -v ./scripts/setup-dev-environment.sh >/dev/null 2>&1; then \
+		./scripts/setup-dev-environment.sh; \
+	else \
+		echo "setup script not found"; \
+	fi
 
 install-deps: ## Install all Python dependencies
```

checkmake (0.3.2)

[warning] Target body for "install-deps" exceeds allowed length of 5 lines (12).

(maxbodylength)